src/ui/screens/CredentialIssuance.tsx

```diff
@@ -108,15 +108,11 @@ export default class CredentialIssuance extends React.Component<Props, State> {
   }
 
   getInviteUrl = async () => {
-    try {
```

Comment:
We might want to rely on the try/catch to detect problems with CORS and internet connectivity in the future. But I agree with moving this to the Agent code!